### PR TITLE
#66 Double click changes account

### PR DESCRIPTION
**Pull request by mohsinshabir1234**
added a new feature to change account when double clicked on any particular account.

### CODE REVIEW

File: gui/control.js [hunk 1]
```diff
@@ -43,14 +48,15 @@ function render() {
...
-            showToast("Error!", errorToMessage[error])
-        });
+        document.getElementById(`switch-btn-${nickname}`).addEventListener('click', async function(){
+            await switchAccountHandler(nickname);
+        }); 
 
-        document.getElementById(`delete-btn-${nickname}`).addEventListener('click', async function () {
+        document.getElementById(`nick-${nickname}`).addEventListener('dblclick', async function () {
+            await switchAccountHandler(nickname);
+        });
```
Comment (adsau59) [inline gui/control.js:57]: use arrow function instead, for better readability and why are we using async/await here?

File: gui/template.js [hunk 2]
```diff
@@ -1,21 +1,22 @@
...
+          </div>
+          
+          <h6 class="text-center">${userNickname}</h6>
+          
+          <button class="btn" id="switch-btn-${userNickname}"><i class="fa-solid  ${isCurrent?'fa-toggle-on':'fa-toggle-off'}"></i></button>
+          <button class="btn" id="delete-btn-${userNickname}"><i class="fa fa-trash"></i></button>
+          <i class="fa fa-check" ${isCurrent?"":"hidden"}></i>
+      </div>
+  </div>
+  `
```
Comment (adsau59) [inline gui/template.js:19]: what is changed here?

File: gui/control.js [hunk 3]
```diff
@@ -17,7 +17,12 @@ const addAccountButton = document.getElementById('add-account-btn');
 window.addEventListener('unload', beforeQuit)
 window.addEventListener('DOMContentLoaded', main)
 
-
+async function switchAccountHandler(nickname) {
+    logger.debug(`switching ${nickname}`)
+    const error = await PSS.switchAccount(nickname);
+    if (!error) return;
+    showToast("Error!", errorToMessage[error])
+};
```
Comment (adsau59) [inline gui/control.js:25]: new line between functions for cleaner code

**Review by adsau59 — CHANGES_REQUESTED** [review]
changes requested

File: gui/template.js (hunk 2 above)
Comment (mohsinshabir1234) [reply gui/template.js:19]: id ="nick=" at line #7

**Review by mohsinshabir1234 — COMMENTED** [review]
#66 changes are done

**Comment by adsau59** [comment]
has other stuff closing pr please create new one without it

**Pull request closed without merging**